packages/python3.rb

```diff
@@ -3,33 +3,48 @@
 class Python3 < Package
   description 'Python is a programming language that lets you work quickly and integrate systems more effectively.'
   homepage 'https://www.python.org/'
-  @_ver = '3.9.2'
+  @_ver = '3.9.4'
   version @_ver
   license 'PSF-2.0'
   compatibility 'all'
-  source_url "https://www.python.org/ftp/python/#{@_ver}/Python-#{@_ver}.tar.xz"
-  source_sha256 '3c2034c54f811448f516668dce09d24008a0716c3a794dd8639b5388cbde247d'
+  source_url "https://www.python.org/ftp/python/#{@_ver}/Python-#{@_ver}.tgz"
```

Comment:
Isn't the `.tar.xz` archive smaller?

Reply:
Yes but the xz file hadn't been uploaded yet when I wanted to build it.

Reply:
I see it's available now.

Reply:
For clarification, the URL was there previously, but it was 404ing. I'll update the URL/hash now that it is actually available.